scripts/build/builders/nxp.py

```diff
@@ -253,11 +321,11 @@ def generate(self):
             cmd += '\nexport ZEPHYR_BASE="$ZEPHYR_NXP_BASE"'
             cmd += '\nunset ZEPHYR_TOOLCHAIN_VARIANT'
 
-            cmd += '\nwest build -p --cmake-only -b {board_name} -d {out_folder} {example_folder}{build_args}'.format(
+            cmd += '\nwest build -p --cmake-only -b {board_name} -d {out_folder} {example_folder} {build_flags}'.format(
                 board_name=self.board.Name(self.os_env),
                 out_folder=self.output_dir,
                 example_folder=self.app.BuildRoot(self.code_root, self.board, self.os_env),
-                build_args=self.WestBuildArgs())
+                build_flags=self.CmakeBuildFlags())
```

Comment:
```suggestion
                build_flags=build_flags)
```

Reply:
Updated the PR with your suggestions, thank you !